src/NLU.DevOps.ModelPerformance.Tests/TestCaseSourceTests.cs

```diff
@@ -403,14 +422,9 @@ public static void ToEntityTestCasesWithFalseNegativeEntityValueMismatch()
 
         [Test]
         [TestCase("foo", "foo", 1, 0, 0, 0)]
-        [TestCase(null, null, 0, 1, 0, 0)]
```

Comment:
Any reason for deleting these test cases rather than fixing them?

Reply:
I should probably add something for null intents and verify that we throw an error.

Reply:
added test validation for null input